### ISSUE TITLE
Aptana3 for Linux crashes on start

### ISSUE THREAD

**Issue by Shoegzer** [aptana/studio3#503]
As of a recent update to Linux Mint 19.1 x64 (possibly the move from Java 10 to Java 11), Aptana now crashes on start.  Looking in the aptana/.metadata/.log file, I see [this](https://pastebin.com/4YpYDFRU).

Researching thie issue elsewhere, the closest I've found is [this](https://github.com/dbeaver/dbeaver/issues/4765) though still not entirely the same symptom.

**Comment by coffeverton** [comment]
Did you get it to work? I'm facing the same problem.

**Comment by Shoegzer** [comment]
No, I haven't unfortunately.  I tried several things and gave up.  This is likely something for the dev team to fix.

**Comment by nikiarsov777** [comment]
I change java's version from 12 to 8 and it ran

**Comment by borneq** [comment]
I have 
$ java --version
openjdk 11.0.5 2019-10-15
OpenJDK Runtime Environment (build 11.0.5+10-post-Ubuntu-0ubuntu1.118.04)
OpenJDK 64-Bit Server VM (build 11.0.5+10-post-Ubuntu-0ubuntu1.118.04, mixed mode, sharing)

and it crash
